examples/enable-operator-and-auto-instrumentation/README.md

```diff
@@ -0,0 +1,189 @@
+kubectl patch deployment vets-service -p '{"spec": {"template":{"metadata":{"annotations":{"instrumentation.opentelemetry.io/inject-java":"false"}}}} }' -n spring-petclinic
+kubectl patch deployment discovery-server -p '{"spec": {"template":{"metadata":{"annotations":{"instrumentation.opentelemetry.io/inject-java":"false"}}}} }' -n spring-petclinic
+kubectl patch deployment visits-service -p '{"spec": {"template":{"metadata":{"annotations":{"instrumentation.opentelemetry.io/inject-java":"false"}}}} }' -n spring-petclinic
+
+# You can verify instrumentation was success on an individual pod wit the
+# following command. If instrumentation was successfull, your application
+# container should have the env variable JAVA_TOOL_OPTIONS set and several
+# OTEL_* env variables to values that are similiar to the output below.
+kubectl describe pod spring-petclinic-9d5bc5fff-5r5gr  -n spring-petclinic
+# Name:             spring-petclinic-9d5bc5fff-5r5gr
+# Namespace:        spring-petclinic
+# Annotations:      instrumentation.opentelemetry.io/inject-java: true
+# Status:           Running
+# Controlled By:  ReplicaSet/spring-petclinic-9d5bc5fff
+# Init Containers:
+#   opentelemetry-auto-instrumentation:
+#     Image:         ghcr.io/open-telemetry/opentelemetry-operator/autoinstrumentation-java:1.23.0
+#     Command:
+#       cp
+#       /javaagent.jar
+#       /otel-auto-instrumentation/javaagent.jar
+#     State:          Terminated
+#       Reason:       Completed
+#       Exit Code:    0
+# Containers:
+#   app:
+#     Image:          ghcr.io/pavolloffay/spring-petclinic:latest
+#     State:          Running
+#       Started:      Mon, 20 Mar 2023 14:26:49 -0600
+#     Ready:          True
+#     Environment:
+#       SPLUNK_OTEL_AGENT:                    (v1:status.hostIP)
+#       OTEL_EXPORTER_OTLP_ENDPOINT:         http://$(SPLUNK_OTEL_AGENT):4317
+#       OTEL_TRACES_EXPORTER:                otlp
+#       JAVA_TOOL_OPTIONS:                    -javaagent:/otel-auto-instrumentation/javaagent.jar
+#       OTEL_SERVICE_NAME:                   spring-petclinic
+#       OTEL_RESOURCE_ATTRIBUTES_POD_NAME:   spring-petclinic-9d5bc5fff-5r5gr (v1:metadata.name)
+#       OTEL_RESOURCE_ATTRIBUTES_NODE_NAME:   (v1:spec.nodeName)
+#       OTEL_PROPAGATORS:                    tracecontext,baggage,b3
+#       OTEL_RESOURCE_ATTRIBUTES:            k8s.container.name=app,k8s.deployment.name=spring-petclinic,k8s.namespace.name=spring-petclinic,k8s.node.name=$(OTEL_RESOURCE_ATTRIBUTES_NODE_NAME),k8s.pod.name=$(OTEL_RESOURCE_ATTRIBUTES_POD_NAME),k8s.replicaset.name=spring-petclinic-9d5bc5fff
+#     Mounts:
+#       /otel-auto-instrumentation from opentelemetry-auto-instrumentation (rw)
+```
+
+</details>
+
+### 7. Check out the results at [Splunk Observability APM](tps://app.signalfx.com/#/apm)
```

Comment:
The realm will be needed for this (and I think us1 is a safer bet if linking to one only).

```suggestion
### 7. Check out the results at [Splunk Observability APM](https://app.signalfx.com/#/apm)
```

Reply:
Good to know, done